ext/phar/dirstream.c

```diff
@@ -260,17 +223,22 @@ static php_stream *phar_make_dirstream(char *dir, HashTable *manifest) /* {{{ */
 			entry[keylen - dirlen - 1] = '\0';
 			keylen = keylen - dirlen - 1;
 		}
-PHAR_ADD_ENTRY:
+		PHAR_ADD_ENTRY:
```

Comment:
We should make a rule for labels, AFAIK Tim you and I all indent differently.

Reply:
To be fair, this was CLion autocorrecting it. But yes a rule would be good.